WebRoot/messages/ZmMsg_fr_CA.properties

```diff
@@ -2142,6 +2142,18 @@ oAuthAppDevice = Appareil
 oAuthAppApprovedDate = Approuv\u00e9
 removeOAuthAppAuthorization = Supprimer l\u2019autorisation de l\u2019application
 oAuthAppAuthorizationRemoveConfirm = Cette action aura pour effet de supprimer l\u2019autorisation accord\u00e9e \u00e0 \u00ab\u00a0{0}\u00a0\u00bb, sur votre \u00ab\u00a0{1}\u00a0\u00bb.
+mobileConFigurationDownloadLabel = T\u00e9l\u00e9charger
```

Comment:
move this to 
WebRoot/messages/ZmMsg_fr.properties

Reply:
Done